blog/2023/12-11-roku-v200/index.mdx

```diff
@@ -0,0 +1,140 @@
+
+Support for Music Videos has been added. If the user has access to the Music Videos library, it will appear in the My Media section and the Latest videos will show on the home view.
+
+### 🏎️ Quickplay Support Expanded To Entire App
+
+Stop wasting time and get your media playing ASAP!
+
+Quickplay support is now across the entire app. Simply highlight an item and press play on your remote. The client will then queue up to 2000 items and start playing.
+
+TV season? Check. Playlist? Check. Your favorite actor? Check. Your music library for a week long rager? Ya' better believe check.
+
+### 📺 TV Show Season Extras Support
+
+![Screenshot showing access to TV show season extras](./Extras.gif)
+
+You can now access TV show season extras in the Jellyfin Roku app. Simply navigate to a season of a TV show and if Jellyfin has found extras, a new button labeled Extras will appear on the left side of the screen under the Shuffle button. Clicking it will toggle between viewing a list of episodes and a list of extras. Simply click on an extra to start watching.
+
+### 📺 TV Episode Version Support
+
+![Screenshot showing access to TV show season extras](./EpisodeVersions.jpg)
+
+Have different versions of a TV episode? You can now select and watch the version you want. On the list of episodes, highlight the episode you want and press * on your remote. The option popup now provides not only audio options, but also version options.
+
+**Note:** If one of the Videos has different audio options, the user must select the desired video version, exit options, then press * again to see the updated audio options.
+
+**Note:** As of this writing, the only way to generate episode versions is in the Android app or using the API.
+
+### 🔍 Search Honors Library Permissions
+
+![Screenshot showing before and after search view](./SearchPermissions.jpg)
+
+Search in Roku now honors library access permissions. If a user doesn't have access to a library, items in that library are no longer returned in search results.
+
+The screenshot above shows what search results look like if a user does not have access to the movie library.
+
+### 🔢 Use Web Client's Home Section Arrangement Setting
+
+![Screenshot of Latest in Music Videos on the home view](./HomeSections.jpg)
+
+A new setting has been added under User Interface > General > Use Web Client's Home Section Arrangement. This setting causes the order of the home sections in the Roku app to match the order of the home sections as set in the server settings.
+
+## ...and on, and on, and on
+
+- Native support for the AV1 codec (no more user setting)
+- Boxsets now sorted by release date by default
+- View your music library by Album Artists
+- Custom subtitle support (required for CJK support) expanded
+- Alpha menu now wraps from bottom to top & top to bottom
+- When connecting to a server, automatically attempt connection using default protocols and ports if not specified
```

Comment:
```suggestion
- When connecting to a new server, automatically attempt connection using default protocols and ports if not specified
```

the new logic is only used when manually connecting to a server from the server select screen. If a connection is made, the new URL is saved in the registry with the protocols and ports included. That way any further connections don't need to infer anything.